custom_components/ramses_cc/water_heater.py

```diff
@@ -96,15 +96,15 @@ def __init__(self, broker, device) -> None:
     def current_operation(self) -> str:
         """Return the current operating mode (Auto, On, or Off)."""
         try:
-            mode = self._device.mode[CONF_MODE]
+            mode = self._device.mode[SZ_MODE]
         except TypeError:
             return
         if mode == ZoneMode.SCHEDULE:
             return STATE_AUTO
         elif mode == ZoneMode.PERMANENT:
-            return STATE_ON if self._device.mode[CONF_ACTIVE] else STATE_OFF
+            return STATE_ON if self._device.mode["active"] else STATE_OFF
```

Comment:
I couldn't find an appropriate constant to import from the library for this.

Reply:
This doesn't look right anyway - is `_device` a 
 - **BdrSwitch** in `ramses_rf.device.heat`, or
 - **StoredHw** in `ramses_rf.system.heat`

Reply:
Woops - its **DhwZone** in `ramses_rf.system.zones`!
```python
if payload[2:4] != "FF":
    result["active"] = {"00": False, "01": True, "FF": None}[payload[2:4]]
```

Reply:
So, I've added:
```python
SZ_ACTIVE: Final[str]  = "active"
```

to `ramses_tx.const` (although that is no good for you today.

Reply:
Perhaps change **manifest.json** to point to the master branch of zxdavb/ramses_rf (something like):
```json
"requirements": [ "git+https://github.com/zxdavb/ramses_rf" ],
```

Reply:
I've got ramses_rf installed from master locally for testing anyway so all good :)